### PR TITLE
Allow to send email to support, with logs

### PR DESCRIPTION
**Pull request by nono**

### CODE REVIEW

**Comment by kosssi** [comment]
this build is red.

**Comment by nono** [comment]
@kosssi it's fixed

File: package.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
     "async": "1.5.2",
     "chokidar": "nono/chokidar#unlink-with-awf-cwd",
     "commander": "2.9.0",
-    "cozy-device-sdk": "1.3.1",
+    "cozy-device-sdk": "nono/cozy-device-sdk#build",
```
Comment (kosssi) [inline package.json]: Can you create a new release on cozy repository?

File: package.json [hunk 2]
```diff
@@ -47,7 +47,7 @@
     "node-uuid": "1.4.7",
     "path-extra": "3.0.0",
     "pouchdb": "5.3.2",
-    "printit": "0.1.18",
+    "printit": "nono/printit#build",
```
Comment (kosssi) [inline package.json]: same for printit?

File: package.json (hunk 1 above)
Comment (nono) [inline package.json]: Well, it depends of a PR on cozy-device-sdk that has not yet been merged.

Comment (kosssi) [inline package.json]: Where is the PR?

Comment (nono) [inline package.json]: I'm wrong. It's the one for printit that has not yet been merged: https://github.com/cozy/printit/pull/20

**Comment by frankrousseau** [comment]
Printit was published as 0.1.20.

**Comment by nono** [comment]
@frankrousseau thanks!

@kosssi maybe you can make a release of cozy-device-sdk too?

**Comment by kosssi** [comment]
You can do it ;)
Add PR with up version and create a tag, travis create the release :dancer:

**Comment by nono** [comment]
@kosssi https://github.com/cozy/cozy-device-sdk/pull/23

**Comment by kosssi** [comment]
https://github.com/cozy-labs/cozy-desktop/pull/300 ;)

File: gui/styles/_two_panes.styl [hunk 3]
```diff
@@ -38,9 +38,11 @@
 
 .two-panes__content
   flex 1
-  overflow hidden
   padding 2em
   color grey-06
+  overflow-x hidden
+  &--dashboard
```
Comment (kosssi) [inline gui/styles/_two_panes.styl]: I don't understand this syntax '&-'

**Pull request merged**